packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -97,6 +110,10 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// Typically an [Image] widget with [Image.fit] set to [BoxFit.cover].
   final Widget? background;
 
+  /// More flexibility build background Widget.
+  final Widget Function(BuildContext context, FlexibleSpaceBarSettings settings,
+      BoxConstraints constraints)? backgroundBuilder;
```

Comment:
Don’t we need more description for this property?
For example, documentation for `background` starts with
```
Shown behind the [title] when expanded.
```